lang/main.json

```diff
@@ -773,5 +774,8 @@
     "lonelyMeetingExperience": {
         "button": "Invite others",
         "youAreAlone": "You are the only one in the meeting"
+    },
+    "helpView": {
+        "header": "Help centre"
```

Comment:
While I love our use of en-GB here, the rest of the app is written in "English Simplified" so -> "center" ?

Please note thatt I take no pleasure in writing this :-P